src/Raven.Client/Documents/Operations/AI/Agents/AiAgentChatTrimmingConfiguration.cs

```diff
@@ -208,20 +208,37 @@ public DynamicJsonValue ToJson()
     /// </remarks>
     public class AiAgentHistoryConfiguration
     {
+        /// <summary>
+        /// Enables history for the AI agents conversations.
+        /// </summary>
+        public AiAgentHistoryConfiguration()
+        {
+            
+        }
+
+        /// <summary>
+        /// Enables history for the AI agents conversations.
+        /// </summary>
+        /// <param name="expiration">The timespan after which history documents expire.</param>
+        public AiAgentHistoryConfiguration(TimeSpan expiration)
+        {
+            HistoryExpirationInSec = (int)expiration.TotalSeconds;
+        }
+
         /// <summary>
         /// The timespan after which history documents expire.
         /// </summary>
         /// <remarks>
         /// History documents are the retained copies of chat messages that have been summarized or truncated.
         /// This property defines how long those history documents will be kept before they are considered expired and eligible for removal.
         /// </remarks>
-        public TimeSpan? HistoryExpiration { get; set; }
+        public int? HistoryExpirationInSec { get; set; }
```

Comment:
Why is that like this and not via time stamp?
What is the meaning of nullable here?

Reply:
To make it easier from other clients?

Reply:
and nullable means you can have history without expiration